pkg/creatives/models.go

```diff
@@ -2,6 +2,25 @@ package creatives
 
 import "time"
 
+type AdCreative struct {
+	ID          string          `json:"creativeID" bson:"_id"`
+	Asset       AdCreativeAsset `json:"asset" bson:"asset"`
+	CreatedAt   time.Time       `json:"createdAt" bson:"createdAt"`
+	Description string          `json:"description,omitempty" bson:"description,omitempty"`
+	ExternalIDs []string        `json:"externalIDs,omitempty" bson:"externalIDs,omitempty"`
+	FileType    string          `json:"fileType,omitempty" bson:"fileType,omitempty"`
+	Format      string          `json:"format,omitempty" bson:"format,omitempty"`
+	Hidden      *bool           `json:"hidden,omitempty" bson:"hidden,omitempty"`
+	SubmittedBy string          `json:"submittedBy,omitempty" bson:"submittedBy,omitempty"`
```

Comment:
yeah, this came from brainstorming with stanford/aaron. there were a couple of reasons for this: one was that we didn't want to risk messing up the programmatic creatives by changing the model (adding fields like submittedBy). we also have a different formula for the creativeID in v2. v1 creativeID is a hash of the bytes of the asset url, which is not always going to exist for non programmatic creatives (like for print creatives). we wanted to move away from the creativeID being a hash and instead use a guid in v2. programmatic had thoughts about eventually moving v1 creatives over to the v2 model with the new creativeID when they're ready. so this v2 AdCreative is really more representative of how we want to universally see creatives instead of just from a programmatic lens.